database/resources/events.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Hackathon",
+    "description": "A hackathon is an event that brings together experts and creates a collaborative environment for solving a certain problem..",
+    "url": "https://tips.hackathon.com/article/what-is-a-hackathon",
+    "category": "resources",
+    "subcategory": "events"
+  },
+  {
+    "name": "Your First Hackathon: A Survival Guide",
+    "description": "Your First Hackathon: A Survival Guide",
```

[Proofreading] The issue here is similar to the one in [line 4](https://github.com/rupali-codes/LinksHub/pull/717/files#r1204750200)
```suggestion
    "description": "Your First Hackathon: A Survival Guide",
```